doc/nomenclature.rst

```diff
@@ -0,0 +1,14 @@
+============
+Nomenclature
+============
+
+Brackets
+--------
+
+ref https://en.wikipedia.org/wiki/Bracket
+
++-------------+----------+--------+
+| ( )         | [ ]      | { }    |
++=============+==========+========+
+| parentheses | brackets | braces |
```

Comment:
i think just calling square brackets, brackets will be confusing

Reply:
See [this comment](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/5915#issuecomment-974674378).